systemdspawner/systemdspawner.py

```diff
@@ -60,6 +60,16 @@ class SystemdSpawner(Spawner):
         help='List of paths that should be marked read-write from the user notebook. Usually used to make a subpath of a readonly path writeable',
     ).tag(config=True)
 
+    unit_extra_properties = List(
```

Comment:
Thank you very much for the patch!

I guess this is a List rather than a Dictionary to preserve order? Does order matter with systemd-run?

Reply:
I don't think order matters, systemd units are declarative.

Reply:
Just to come back to this - ordering *does* matter in some cases - see https://www.freedesktop.org/software/systemd/man/systemd.service.html#ExecStartPre= for example.